scripts/train/yamls/finetune/mpt-7b_dolly_sft.yaml

```diff
@@ -0,0 +1,125 @@
+max_seq_len: 2048
+global_seed: 17
+
+# Run Name
+run_name: # If left blank, will be read from env var $COMPOSER_RUN_NAME
+
+model:
+  name: hf_causal_lm
+  device: cpu
+  pretrained: true
+  pretrained_model_name_or_path: mosaicml/mpt-7b
+  config_overrides:
+    attn_impl: triton
+
+# Tokenizer
+tokenizer:
+  name: mosaicml/mpt-7b
+  kwargs:
+    model_max_length: ${max_seq_len}
+
+
+# Dataloaders
+train_loader:
+  name: finetuning
+  dataset:
+    name: mosaicml/dolly_hhrlhf
+    split: train
+    tokenizer_name: ${tokenizer_name}
+    max_seq_length: ${max_seq_len}
+    allow_pad_trimming: false
+    decoder_only_format: true
+    # Use `python llmfoundry/data/packing.py --yaml-path /path/to/this/yaml/ ...`
+    # to profile this run's optimal packing_ratio as it depends on GPU count,
+    # batch size, sequence length
+    # packing_ratio:
+    shuffle: true
+  drop_last: true
+  num_workers: 8
+  pin_memory: false
+  prefetch_factor: 2
+  persistent_workers: true
+  timeout: 0
+
+eval_loader:
+  name: finetuning
+  dataset:
+    name: mosaicml/dolly_hhrlhf
+    split: test
+    tokenizer_name: ${tokenizer_name}
+    max_seq_length: ${max_seq_len}
+    allow_pad_trimming: false
+    decoder_only_format: true
+    # packing_ratio:
+    shuffle: true
+  drop_last: true
+  num_workers: 8
+  pin_memory: false
+  prefetch_factor: 2
+  persistent_workers: true
+  timeout: 0
+
+# Optimization
+scheduler:
+  name: linear_decay_with_warmup  # linear no warmup is HF default which dolly used
+  t_warmup: 50ba  # add some warmup though, seems to help with MPT
+  alpha_f: 0
+
+optimizer:
+  # Based on Dolly
+  name: decoupled_adamw
+  lr: 5.0e-6
+  betas:
+  - 0.9
+  - 0.999
+  eps: 1.0e-8
+  weight_decay: 0
+
+algorithms:
+  gradient_clipping:
+    clipping_type: norm
+    clipping_threshold: 1.0
+
+  max_duration: 2ep  # 2-3 epochs seems like the sweet spot
```

Comment:
I think this block is accidentally indented (i.e. its under `algorithms` right now